block/builder_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package block
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vechain/thor/v2/thor"
+	"github.com/vechain/thor/v2/tx"
+)
+
+func TestBuilder(t *testing.T) {
```

Comment:
Apologies, yeah the table test might be overkill, I might have mislead you here. 

This test seems to be testing out the set and get for the builder, it can probably be simplified into something like the below ?

```
builder := &Builder{}

id := thor.Bytes32{1, 2, 3}
builder.ParentID(id)

ts := uint64(1234567890)
builder.Timestamp(ts)

score := uint64(100)
builder.TotalScore(score)
...
assert.Equal(t, id, b.header.ParentID())
assert.Equal(t, ts, b.header.Timestamp())
assert.Equal(t, score, b.header.TotalScore())
```